app/src/main/java/org/thoughtcrime/securesms/conversation/disappearingmessages/DisappearingMessagesViewModel.kt

```diff
@@ -59,16 +59,28 @@ class DisappearingMessagesViewModel(
     init {
         viewModelScope.launch {
             val expiryMode = storage.getExpirationConfiguration(threadId)?.expiryMode ?: ExpiryMode.NONE
-            val recipient = threadDb.getRecipientForThreadId(threadId)
-            val groupRecord = recipient?.takeIf { it.isClosedGroupRecipient }
+            val recipient = threadDb.getRecipientForThreadId(threadId)?: return@launch
+            val groupRecord = recipient.takeIf { it.isLegacyClosedGroupRecipient || it.isClosedGroupV2Recipient }
```

Comment:
This OR statement can be replaced by the new shorthand `isGroup` after you have renamed the existing methods